lib/pages/chat/chat.dart

```diff
@@ -1449,6 +1460,12 @@ class ChatController extends State<Chat>
     );
   }
 
+  void hideKeyBoard() {
+    if (keyboardVisibilityController.isVisible || inputFocus.hasFocus) {
```

Comment:
we already have Global hide keyboard funciton
`dismissKeyboard`

Reply:
1. If I want to unfocus a specific widget that you have direct control over (e.g., a text field in a form), should create a FocusNode for that widget and use `inputFocus.unfocus()`.
2. If I need to unfocus a widget from a different part of your application or handle global focus management, you can use FocusManager.instance.primaryFocus?.unfocus();.

In this case I want to choose option 1